vlttng/venv.py

```diff
@@ -547,14 +547,18 @@ def _build_lttng_tools(self):
         if project is None:
             return
 
-        if '--with-lttng-ust-prefix' in project.configure or '--without-lttng-ust' in project.configure:
```

Comment:
Use the named list here:

    for opt in lttng_ust_opts:
        if opt in project.configure:
            # ...